spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -369,6 +370,15 @@ public Session getSession() {
 		 */
 		private int acceptCount = 100;
 
+		/**
+		 * The maximum number of idle processors that will be retained in the cache and
```

Comment:
Thank you but we're not there yet. Could you please look at other keys in `ServerProperties` for inspiration? Key description does not start with `a` or `the` and can't contain javadoc tags as these are not processed by the annotation processed. We don't document default value either as the value is already set there and made available in the metadata.

Looking at your update, this looks ideal to me:

> Maximum number of idle processors that will be retained in the cache and reused with a subsequent request.

Please note that key descriptions are not meant to copy/paste the official documentation of the thing it's configuring.

Reply:
ok, thx for advice